apps/snitch_core/lib/core/data/schema/stock/stock_transfer.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Snitch.Data.Schema.StockTransfer do
+  @moduledoc """
+  Model to track inventory transfer in location
+  """
+  use Snitch.Data.Schema
+  use Snitch.Data.Schema.Stock
+
+  @type t :: %__MODULE__{}
+
+  schema "snitch_stock_transfers" do
+    field(:type, :string)
+    field(:reference, :string)
+    field(:number, :string)
+
+    belongs_to(:source_location, StockLocation)
+    belongs_to(:destination_location, StockLocation)
+
+    timestamps()
+  end
+
+  @create_fields ~w(number destination_location_id)a
+
+  def create_fields, do: @create_fields
+
+  @doc """
+  Stock Movements and Stock Transfers are only created.
+  Inorder to update them we can create another stock movement to reverse its effects.
```

Comment:
Does that mean the effects of a `StockTransfer` can be negated by a `StockMovement`? But a transfer has neither stock_item nor quantity. :confused: